app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [OPTIONAL] Same thing goes for this too as I mentioned for the post. Just like we have for the post, `after_save` 
`update_comments_counter`